### DISCUSSION TITLE
Do we need rn-nodeify hack to use walletConnect via web3-react in a react native project?

### DISCUSSION THREAD

**Discussion by manosamy** [NoahZinsmeister/web3-react#372] [Q&A]
for a react-native project, should we install rn-nodeify as suggested here, if we were to use walletConnect via web3-react?
https://www.npmjs.com/package/@walletconnect/react-native-dapp/v/1.7.2-rc.0

**Comment by manosamy** [comment]
For additional context, the command "npx rn-nodeify --install --hack" installs packages like readable-stream, stream-browserify, timers-browserify, url, vm-browserify, react-native-tcp, react-native-udp, and so on...

**Reply by nishilshah17** [reply]
Did you manage to get it working? I'm trying to use walletconnect connector now on react native and when I call activate function nothing happens.
